_pages/ide/offline.md

```diff
@@ -17,7 +17,7 @@ Follow the instructions for your Linux distribution to [install Docker Engine](h
 1. Open up a terminal window and run the following command to create a new CS50 IDE instance:
 
     ```
-    docker run --privileged -e "OFFLINE_IP=127.0.0.1" -e "OFFLINE_PORT=8080" --name ide50 -d -p 5050:5050 -p 8080-8082:8080-8082 cs50/ide50-offline
+    docker run --privileged -e "OFFLINE_IP=127.0.0.1" -e "OFFLINE_PORT=8080" --name ide50 -d -p 5050:5050 -p 8080-8082:8080-8082 cs50/ide
```

Comment:
Mind reminding me...?

1. What's the difference between `OFFLINE_PORT` and `5050` here?
1. How is `OFFLINE_IP` used exactly?

Reply:
`5050` is the port that the Cloud9's server listens on. `OFFLINE_PORT` is the default port used by Cloud9 (e.g., in plugins) for different applications. `OFFLINE_IP` is used as `C9_HOSTNAME`.

Reply:
Could we simplify to just `IP` and `PORT` in that case? And could/should we simplify this further by shipping an `ide50` command that folks can `pip` install that does all this for folks?